### PR TITLE
[20] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by MYKEL7**
## Task: `Sign-Up Form Validation`
- `Validated the signUp page`

### CODE REVIEW

File: submissions/onyemeMichael/htmlForms/signUp.html [hunk 1]
```diff
@@ -75,11 +90,10 @@ <h3>Already got an account?<a href="#" class="log-in" >Sign in</a></h3>
             <p class="kt">Sign up for our Newsletter</p>
             <div class="newsletter">
                 <form>
-                    <input type="email" name="email" id="email" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
+                    <input type="email" name="email" id="let" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
```
Comment (ToluAkin) [inline submissions/onyemeMichael/htmlForms/signUp.html:93]: @MYKEL7 Move the style here to the css file

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/onyemeMichael/htmlForms/signUp.html (hunk 1 above)
Comment (MYKEL7) [reply submissions/onyemeMichael/htmlForms/signUp.html]: Done.

**Comment by MYKEL7** [comment]
Done.

File: submissions/onyemeMichael/css/contact.css [hunk 2]
```diff
@@ -202,6 +202,12 @@ body {
      font-size: 18px;
  }
 
+ #let {
+   width: 50%;
+   border-radius: 0px;
+   padding: 5px;
```
Comment (adaugochi) [inline submissions/onyemeMichael/css/contact.css:208]: Remove `px` on zero value

**Review by adaugochi — COMMENTED** [review]
Fix code factor comments

File: submissions/onyemeMichael/html/product.html [hunk 3]
```diff
@@ -78,7 +78,7 @@ <h5>Price: &#8358;<strong>88,000</strong></h5>
         <h3 class="kt">Sign up for our Newsletter</h3>
         <div class="newsletter">
             <form>
-                <input type="email" name="email" id="email" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
+                <input type="email" name="email" id="let" placeholder="Enter your email address">
```
Comment (ToluAkin) [inline submissions/onyemeMichael/html/product.html:81]: Why are you changing the id of the email input field to `let`? @MYKEL7 I don't think the new `id` is the right name to use.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**